Python/math/Area.py

```diff
@@ -61,13 +61,42 @@ def area_circle(radius):
 
 def main():
     print("Areas of various geometric shapes: \n")
-    print(f"Rectangle: {area_rectangle(10, 20)=}")
-    print(f"Square: {area_square(10)=}")
-    print(f"Triangle: {area_triangle(10, 10)=}")
-    print(f"Parallelogram: {area_parallelogram(10, 20)=}")
-    print(f"Trapezium: {area_trapezium(10, 20, 30)=}")
-    print(f"Circle: {area_circle(20)=}")
+    
```

Comment:
It would be better if you add a menu here and ask user to choose the shape whose area they want to calculate so that you could call the functions and display the output accordingly.

Reply:
yes i have done that change and am submiitng another pr